### PR TITLE
Character

### PR DESCRIPTION
**Pull request by jakezehring**

### CODE REVIEW

File: www/templates/character.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          </label>
+          <label class="item item-input">
+              <input type="text" placeholder="Race">
+          </label>
+          <label class="item item-input">
+              <textarea placeholder="Class"></textarea>
+          </label>
+          <label class ="item item-input">
+              <input type="text" placeholder="skill rank" />
+              <input type="text" placeholder="level" />
```
Comment (Nickel-Less) [inline www/templates/character.html]: Nice work! However I noticed you used input type="text" for everything, did you know you can do a lot with that tag? here is the link for all the avaliable types: http://www.w3schools.com/tags/att_input_type.asp

**Comment by Nickel-Less** [comment]
I do not know if the changes didn't get pushed up, but as of now this branch is not able to be merged

**Comment by jakezehring** [comment]
Mb I forgot to sync try it again now

**Comment by jakezehring** [comment]
Mb I forgot to sync try it again now

**Comment by Nickel-Less** [comment]
Looks good!

**Pull request merged**